### DISCUSSION TITLE
[BP#28] Utiliser les commentaires conditionnel

### DISCUSSION THREAD

**Discussion by tbroyer** [cnumr/best-practices#118] [Bonnes pratiques] [answered]
Ces commentaires ne sont plus supportés depuis IE10, je pense qu'on peut sereinement supprimer cette BP pour obsolescence :wink:

**Comment by ACTLEM** [comment]
D'accord avec toi. IE tout confondu est à moins de 1% de part de marché dans le monde, donc je n'ose imaginer pour IE 10 ^^

**Comment by arnolem** [comment]
On est D'ACCORD 😉

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/130
